src/batch/src/executor/aggregation/orderby.rs

```diff
@@ -77,7 +77,7 @@ impl ProjectionOrderBy {
     fn push_row(&self, state: &mut State, row: RowRef<'_>) -> Result<()> {
         let key =
             memcmp_encoding::encode_row(row.project(&self.order_col_indices), &self.order_types)
-                .map_err(|e| ExprError::Internal(anyhow!("failed to encode row, error: {}", e)))?;
+                .context("failed to encode row")?;
```

Comment:
Use `Anyhow::Context::context` to create a new internal error with the given message, while keeping the error source.